lib/mmbooking_Clone/admin/booking.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Mmbooking_Clone.Admin.Booking do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "bookings" do
+    field :date, :date
+    field :has_visited, :string
+    field :seats_available, :string
+    field :session, :string
```

Comment:
no need this column and association missing:

add column group_name and status